tools/CI/README_Metadata_SampleSyncCheck/sample_sync.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import subprocess
+import os
+
+def main():
+    '''
+    Usage: python sample_sync.py 
+    '''
+    # args = sys.argv[1]
+    
+    readme_script_path = os.path.join(".","tools", "readme_copy", "readme_copy.py")
+    metadata_script_path = os.path.join(".", "tools", "metadata_tools", "process_metadata.py")
+
+    print("Copying readmes")
+    subprocess.run(["python", readme_script_path])
+
+    print("Updating metadata")
+    subprocess.run(["python", metadata_script_path, "src"])
```

Comment:
```suggestion
    subprocess.run(["python", metadata_script_path])
```
IIRC the argument wasn't actually necessary here, something else was the problem at the time.